sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.cs

```diff
@@ -542,5 +543,51 @@ public override void Dispose()
 
             public override string ToString() => _responseMessage.ToString();
         }
+#if NETCOREAPP
+        private static SocketsHttpHandler ApplyOptionsToHandler(SocketsHttpHandler httpHandler, HttpPipelineTransportOptions? options)
+        {
+            if (options == null)
+            {
+                return httpHandler;
+            }
+
+            // ServerCertificateCustomValidationCallback
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER")))
+            {
+                if (options.ServerCertificateCustomValidationCallback != null)
+                {
+                    httpHandler.SslOptions.RemoteCertificateValidationCallback = (_, certificate, _, _) =>
```

Comment:
What are the parameters we are ignoring? Is it possible someone might care about them? Do we need an EventArg type for future-proofing?

Reply:
Agreed. Will look into this a bit more.